modules/pool/detail/components/stats/PoolOverallStats.tsx

```diff
@@ -42,8 +38,14 @@ export default function PoolOverallStats() {
         sumBy(
             pool.staking?.farm?.rewarders || [],
             (rewarder) => priceFor(rewarder.tokenAddress) * parseFloat(rewarder.rewardPerSecond) * 86400,
+        ) +
+        sumBy(
+            pool.staking?.gauge?.rewards || [],
+            (reward) => priceFor(reward.tokenAddress) * parseFloat(reward.rewardPerSecond) * 86400,
         );
 
+    const rewards = pool.staking?.farm?.rewarders || pool.staking?.gauge?.rewards;
```

Comment:
But here you do an OR, saying that only one could exist. The two operations are at odds with eachother, pick a direction